test/TestDataFactory.js

```diff
@@ -0,0 +1,68 @@
+// ## TestDataFactory functions
+// Emulate old, pre-RDF/JS output (plain strings instead of Terms) to make tests pass
+
+var XSD  = 'http://www.w3.org/2001/XMLSchema#';
+
+const xsd = {
+    decimal: XSD + 'decimal',
+    boolean: XSD + 'boolean',
+    double:  XSD + 'double',
+    integer: XSD + 'integer',
+    string:  XSD + 'string',
+  };
+
+// ### Creates an IRI
+function namedNode(iri) {
+  return iri;
+}
+
+// ### Creates a blank node
+function blankNode(name) {
+  return '_:b' + name;
+}
+
+// ### Creates a literal
+function literal(value, languageOrDataType) {
+  // Create a language-tagged string
+  if (typeof languageOrDataType === 'string')
+    return '"' + value + '"@' + languageOrDataType.toLowerCase();
+
+  // Create a datatyped literal
+  var datatype = languageOrDataType && languageOrDataType.value || '';
+  if (!datatype) {
+    switch (typeof value) {
+    // Convert a boolean
+    case 'boolean':
+      datatype = xsd.boolean;
+      break;
+    // Convert an integer or double
+    case 'number':
+      if (Number.isFinite(value))
+        datatype = Number.isInteger(value) ? xsd.integer : xsd.double;
+      else {
+        datatype = xsd.double;
+        if (!Number.isNaN(value))
+          value = value > 0 ? 'INF' : '-INF';
+      }
+      break;
```

Comment:
We probably shouldn't be too smart here. RDF/JS doesn't do conversion either, so it's up to the parser to call this with a string.

Reply:
Yeah this part needs to at looked after the datatype/lang issue is resolved.
Probably just removed, I copy-pasted it from N3.js.